srcpkgs/zam-plugins/files/zam-plugins.menu

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE Menu PUBLIC "-//freedesktop//DTD Menu 1.0//EN" "http://www.freedesktop.org/standards/menu-spec/menu-1.0.dtd">
```

Comment:
And this one? Vendoring in such stuff doesn't make sense, improve upstream to make them ship this stuff instead.

Reply:
See https://github.com/void-linux/void-packages/pull/31665#discussion_r659342213

Reply:
Same comment applies.